modules/integration/tests-integration/pom.xml

```diff
@@ -277,6 +277,13 @@
             <artifactId>wso2xfer</artifactId>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.client.configcontext.provider</artifactId>
+            <version>1.0.0</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/src/test/lib/org.wso2.carbon.client.configcontext.provider-1.0.0.jar</systemPath>
```

Comment:
Since kernel 4.4.0 is shipped with config context provide implementation, we need to use that.